west.yml

```diff
@@ -203,7 +203,7 @@ manifest:
       groups:
```

Comment:
Please create a better description for this PR. When it will be merged there won't be any reference to PR497.

So please say something along the lines: update hal_nxp to push support for imx91.

Reply:
yes, will update with the info, actually it will be updated to the hal_nxp SHA when that PR is merged.